geomstats/geometry/minkowski.py

```diff
@@ -16,51 +17,40 @@ class Minkowski(Manifold):
     """
 
     def __init__(self, dim):
-        super(Minkowski, self).__init__(dim=dim)
+        super(Minkowski, self).__init__(dim=dim,
+                                        ambient_manifold=Euclidean(dim))
```

Comment:
Should we rename Euclidean in VectorSpace? or Have a VectorSpace class?

Reply:
Maybe yes

Reply:
Minkowski space is really just Euclidean with a different metric, we should have something to create that class without actually writing a new class